personio-personnel-data-api-oa3.yaml

```diff
@@ -1934,7 +1934,7 @@ paths:
         - name: end_date
           in: query
           required: false
-          description: Filter by absence periods that start at or before given date.
+          description: Filter by absence periods that start at or after given date.
```

Comment:
question: Is this an intentional change? It should be *before*, right?

Reply:
@rajki, I fixed the description. It's a bit confusing in terms of which periods are included 😅